openmrs/src/main/java/org/motechproject/openmrs/domain/ProgramEnrollment.java

```diff
@@ -32,6 +35,8 @@
     @Expose
     private List<StateStatus> states;
 
+    private String enrolled = ENROLLED;
```

Comment:
please add here boolean field and additionally add getter which returns String for task provider